packages/esp32-idf-headers.3.0/opam

```diff
@@ -1,8 +1,9 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "Lucas Pluvinage <lucas.pluvinage@ens.fr>"
 authors: ["Lucas Pluvinage <lucas.pluvinage@ens.fr>"]
 depends: [
-"gcc-toolchain-esp32"
+  "gcc-toolchain-esp32" { os = "linux" }
+  "macos-toolchain-esp32" { os != "linux" }
```

Comment:
Is it possible to have something more precise than `{ os != "linux" }` for macos ? I wouldn't want windows people to download this inadvertently !

Looks like `os = "macos"` should do the trick.